src/AutoRest.CSharp/Common/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -136,6 +136,7 @@ internal static string SaveConfiguration()
                     WriteIfNotDefault(writer, Configuration.Options.SkipCSProjPackageReference, Configuration.SkipCSProjPackageReference);
                     WriteIfNotDefault(writer, Configuration.Options.Generation1ConvenienceClient, Configuration.Generation1ConvenienceClient);
                     WriteIfNotDefault(writer, Configuration.Options.SingleTopLevelClient, Configuration.SingleTopLevelClient);
+                    WriteIfNotDefault(writer, Configuration.Options.KeepUnusedModels, Configuration.KeepUnusedModels);
```

Comment:
Can we have this config support 3 values?

1. Don't change accessor
2. Make accessor internal
3. Delete file from library

Default being 3.  There are some debug use cases where we might want to keep the files internal and not remove them.  By default we do want to remove them to optimize package sizes.

Reply:
Good idea. Therefore we are making something like the level we would like to keep the models?
I would like to reverse this order to make the default value to be `0`, like:
0 - Delete unused files from library (default)
1 - Make accessor internal
2 - Do nothing
What do you think? Or should we use strings instead of integer on this?

Reply:
Well, I refactored this configuration and made it an enumeration of three values:
1. RemoveAll (default, which will remove all unused models)
2. Internalize (only make unused models internal, but keep them)
3. KeepAll (do nothing)